ecosystem/typescript/sdk/src/aptos_types/abi.ts

```diff
@@ -30,7 +30,10 @@ export class ArgumentABI {
    * @param name
    * @param type_tag
    */
-  constructor(public readonly name: string, public readonly type_tag: TypeTag) {}
+  constructor(
+    public readonly name: string,
+    public readonly type_tag: TypeTag,
```

Comment:
Are these coming from pnpm fmt? The comma after the last param seems weird lol

Reply:
Yes, this comes from the formatter directly.  Commas are optional in most places, except JSON, where they're explicitly not allowed